android/src/main/java/com/evollu/react/fcm/FIRMessagingModule.java

```diff
@@ -54,12 +60,38 @@ public void requestPermissions(){
     }
 
     @ReactMethod
+    public void getInitFCMData(Promise promise) {
```

Comment:
when app is in background, notification comes in "onNotificationReceived" event. It is to be consistent with the behavior in iOS